### ISSUE TITLE
On Importing a app via zip file, Unable to pick old Entities

### ISSUE THREAD

**Issue by Abhishek-Choudharyy** [wit-ai/wit#1092]
I just want to clone an app. So, I export data from the old app and Import that zip file into a new app. The new app is unable to extract entities (These entities are already trained in the old app). Let me know the best way to clone as I don't want to make100+ entities again and again

**Comment by patapizza** [comment]
Hi @Abhishek-Choudharyy,

Please use the provided template - what are your app ids (old and new one)?

**Comment by Abhishek-Choudharyy** [comment]
Old App ID : 5ab4a460-3708-42c4-a787-b4ad903f2bf4
New App ID : 5ae1885f-0c24-461f-b535-5aedca248d8c

**Comment by rantoniops** [comment]
I'm facing the same problem.

I create a new app by importing the zip from an old app with trained entities but the new app isn't recognizing any of them even though the intent does show as imported.

Old app id: 5af9f9b9-c261-45d4-b08f-a187aed9ffeb
New app id: 5b080cd4-a513-462b-8ddf-479e0048bec4

**Comment by patapizza** [comment]
@Abhishek-Choudharyy Can you share a few examples that don't work with the new app, but work with the old one?

<img width="971" alt="screen shot 2018-05-25 at 1 51 01 pm" src="https://user-images.githubusercontent.com/1015208/40558886-c30f42d8-6022-11e8-941d-93801260d2cd.png">

@rantoniops Your app should work now, it was stuck in training.

<img width="977" alt="screen shot 2018-05-25 at 1 54 00 pm" src="https://user-images.githubusercontent.com/1015208/40558968-20768d78-6023-11e8-8552-7141ece4eeb7.png">

**Comment by Abhishek-Choudharyy** [comment]
@patapizza Please check in this new app id : 5ae188c5-54cd-4fa4-9859-aa195c2267aa

1) "Are coffee pots in the room? " is not working in this app
2) It is not picking keywords like "gym, yoga " in this app.
This app is cloned from the old app : 5ab4a460-3708-42c4-a787-b4ad903f2bf4

**Comment by patapizza** [comment]
@Abhishek-Choudharyy: The first issue is resolved. `Amenities_health` doesn't have any validated samples, so it can't pick up the keywords.

<img width="1002" alt="screen shot 2018-05-25 at 4 55 12 pm" src="https://user-images.githubusercontent.com/1015208/40565953-84bc7b8a-603c-11e8-8d6d-6726659a9c95.png">

**Comment by Abhishek-Choudharyy** [comment]
First of all gym, yoga is part of Amenities_health whose lookup strategy is: keywords.
![screenshot from 2018-05-26 11-30-56](https://user-images.githubusercontent.com/29659840/40573044-5b8c9992-60d8-11e8-8ead-310b90f090c9.png)
But it is unable to pick the keyword.

I imported this app around 20 days ago.The training part is done on 26th of may only after raising an issue here.
![screenshot from 2018-05-26 11-29-34](https://user-images.githubusercontent.com/29659840/40573051-a460ed1c-60d8-11e8-808d-36204c45ce64.png)

How much does it take to train an app normally??

**Comment by patapizza** [comment]
@Abhishek-Choudharyy Have you tried validating samples for your entity?
@rantoniops 2- Yes I did; validating new samples should queue a new training.

Please follow https://github.com/wit-ai/wit/issues/1063 re training status and import/export. Closing up as duplicate of that.

**Comment by Abhishek-Choudharyy** [comment]
@patapizza 
1) I am importing an app from old app. If we have to validate samples again for entities(already present in the old app).then what's the use of cloning?? 
Let me know if it is compulsory to validate sample for all entities (although these entities are already validated in the old app)

2) I am using samples API ( https://wit.ai/docs/http/20170307#post--samples-link ) to upload samples Last time, I uploaded samples in the app (app id: 5ae188c5-54cd-4fa4-9859-aa195c2267aa) on 2-4th of May. The app was only get trained (Trained on 26th May )after creating the issue here. How much time app will take to show results after uploading data.

**Comment by patapizza** [comment]
@Abhishek-Choudharyy Again, we are aware of the issue and it's being tracked in #1063. Validating one sample is enough.
